src/main/java/hudson/plugins/git/browser/TFS2013GitRepositoryBrowser.java

```diff
@@ -82,7 +82,7 @@ private GitSCM getScmFromProject(GitChangeSet changeSet) {
     public static class TFS2013GitRepositoryBrowserDescriptor extends Descriptor<RepositoryBrowser<?>> {
 
         public String getDisplayName() {
-            return "Microsoft TFS 2013";
+            return "Microsoft Team Foundation Server";
```

Comment:
Does this also work with [Visual Studio Team Services (VSTS)](https://www.visualstudio.com/products/visual-studio-team-services-vs)?  If so, maybe we would want the string to be `Microsoft TFS/VSTS`?